tests/operations/test_operations.py

```diff
@@ -51,3 +53,22 @@ def test_bitwise_lshift(a, b):
 )
 def test_bitwise_rshift(a, b):
     return a >> b
+
+
+@mt(
+    run(np.arange(10), result=np.prod(np.arange(10))),
```

Comment:
The arange includes zero, so the result is going to be zero. I'm not sure if that's what you meant to do.

Reply:
Indeed, I will use `np.arange(1, 11)` instead and hardcode the expected result value.